app/layout.tsx

```diff
@@ -77,9 +77,38 @@ export default async function RootLayout({
   return (
     <html lang="en">
       <head>
+        <script>
+          window.dataLayer = window.dataLayer || [];
+          {/* <!-- Google Tag Manager --> */}
```

Comment:
There might be a typo in the code snippet. I think there should be a closing `</script>` here, and not on line 97. You can't have nested script tags and the code snippet that is given looks wrong. We can verify tomorrow.

Reply:
there's a script tag at the top of the first code snipped in the ticket https://newyorkpubliclibrary.atlassian.net/jira/software/c/projects/DR/boards/127?selectedIssue=DR-3434

Reply:
updated. should be correct now.